content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 3
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
+
+## Prerequisites
+
+1. [Download the Istio release] (/docs/setup/getting-started/#download)
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/)
+1. Check the requirements [for Pods and Services](/docs/ops/deployment/requirements/)
+1.  Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, or alternatively, every pod in the mesh via L3 networking.
+
+## Installing the Istio control plane
+
+1. Generate intermediate CA certificates for each cluster's CA from your
+    organization's root CA. The shared root CA enables mutual TLS communication
+    across different clusters or virtual machines. For illustration purposes,
+    the following instructions use the certificates from the Istio samples
+    directory for the cluster.
+
+1. Run the following commands on each cluster in the mesh where a virtual
+    machine connection should be established. See
+    [Certificate Authority (CA) certificates](/docs/tasks/security/cert-management/plugin-ca-cert/)
+    for more details on configuring an external CA.
+
+    {{< text bash >}}
+    $ cd samples/certs
+    $ make vm
```

Comment:
this should be `make vm-certs-wl`

Reply:
this is resolved although the updated workflow could use a new test.